nucleus/payara-modules/asadmin-audit/src/main/java/fish/payara/audit/admin/SetAdminAuditServiceNotifierConfiguration.java

```diff
@@ -0,0 +1,222 @@
+import org.glassfish.api.admin.AdminCommandContext;
+import org.glassfish.api.admin.ExecuteOn;
+import org.glassfish.api.admin.RestEndpoint;
+import org.glassfish.api.admin.RestEndpoints;
+import org.glassfish.api.admin.RuntimeType;
+import org.glassfish.api.admin.ServerEnvironment;
+import org.glassfish.config.support.CommandTarget;
+import org.glassfish.config.support.TargetType;
+import org.glassfish.hk2.api.PerLookup;
+import org.glassfish.internal.api.Target;
+import org.jvnet.hk2.annotations.Service;
+import org.jvnet.hk2.config.ConfigSupport;
+import org.jvnet.hk2.config.SingleConfigCode;
+import org.jvnet.hk2.config.TransactionFailure;
+
+import com.sun.enterprise.config.modularity.ConfigModularityUtils;
+import com.sun.enterprise.config.serverbeans.Config;
+import com.sun.enterprise.util.SystemPropertyConstants;
+
+import fish.payara.audit.AdminAuditConfiguration;
+import fish.payara.audit.AdminAuditService;
+
+import fish.payara.nucleus.notification.configuration.NotificationServiceConfiguration;
+import fish.payara.nucleus.notification.configuration.Notifier;
+import fish.payara.nucleus.notification.configuration.NotifierConfiguration;
+import fish.payara.nucleus.notification.configuration.NotifierConfigurationType;
+import fish.payara.nucleus.notification.configuration.NotifierType;
+
+/**
+ * Admin command to set enabled and noisy flags for a specific {@link Notifier} configuration as part of the
+ * {@link AdminAuditConfiguration}.
+ * 
+ * This should not be confused with the {@link NotifierConfiguration} itself for that specific {@link Notifier} that
+ * exists as part of {@link NotificationServiceConfiguration}.
+ * 
+ * When {@link #noisy} parameter is not specified the according {@link NotifierConfiguration} is loaded from the
+ * {@link NotificationServiceConfiguration} to use its {@link NotifierConfiguration#getNoisy()} as automatic fallback
+ * setting.
+ *
+ * @author jonathan coustick
+ * @since 5.192
+ */
+@Service(name = "set-admin-audit-service-notifier-configuration")
+@PerLookup
+@ExecuteOn({RuntimeType.DAS, RuntimeType.INSTANCE})
+@TargetType(value = {CommandTarget.DAS, CommandTarget.STANDALONE_INSTANCE, CommandTarget.CLUSTER, CommandTarget.CLUSTERED_INSTANCE, CommandTarget.CONFIG})
+@RestEndpoints({
+        @RestEndpoint(configBean = AdminAuditConfiguration.class,
+                opType = RestEndpoint.OpType.POST,
+                path = "set-admin-audit-service-notifier-configuration",
+                description = "Sets the Configuration for specific Notifier for the Admin Audit Service")
+})
+public class SetAdminAuditServiceNotifierConfiguration implements AdminCommand {
+
+    @Inject
+    private Target targetUtil;
+
+    @Inject
+    private ServerEnvironment server;
+
+    @Inject
+    private ConfigModularityUtils configModularityUtils;
+
+    @Inject
+    private AdminAuditService auditService;
+
+    @Param(name = "dynamic", optional = true, defaultValue = "false")
```

Comment:
As parameters are the public interface of a command and injections are somewhat more implementation details, can you start the command with all the @Param annotated instance fields?